apicast/bin/apicast

```diff
@@ -110,6 +110,7 @@ done
 
 main+=("daemon ${daemon};")
 main+=("worker_processes ${worker_processes};")
+main+=$(printenv | awk '$1 ~ /^(APICAST|THREESCALE)_/ {split($0,env,"="); print "env", env[1] ";"}')
```

Comment:
@mayorova want to create some checkboxes todo in the description what needs to be done for this to be merged?

1. I don't see anywhere in the documentation that we would suggest using nginx to start by pointing to a configuration. Not sure why anyone would do that. Also not sure how many people are using v2 already (but close to 0).

2. We can't **make sure** it stays in sync unless we are able to code it. Reviews fail and docs get out of sync. Proposals welcome. Tests are a good way to make sure the documentation works (and become the documentation).